src/showingpreviously/cinemas/showcase.py

```diff
@@ -0,0 +1,169 @@
+import json
+import re
+
+from datetime import datetime
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE
+
+
+CINEMAS_URL = 'https://www.showcasecinemas.co.uk'
+SHOWINGS_API_URL = 'https://movieapi.showcasecinemas.co.uk/movies/45/{cinema_id}?expandGenres=true&splitByAttributes=true'
+
+CINEMAS_LIST_PATTERN = re.compile(r'\'list\':\s*(?P<cinemas_list>\[.+?])\s*}')
+
+CHAIN = Chain('Showcase Cinemas')
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def get_cinemas_as_dict() -> dict[str, Cinema]:
+    r = get_response(CINEMAS_URL)
+    cinemas_str = CINEMAS_LIST_PATTERN.search(r.text).group('cinemas_list')
+    try:
+        cinemas_data = json.loads(cinemas_str)
+    except json.JSONDecodeError:
+        raise CinemaArchiverException(f'Error decoding JSON data from URL {CINEMAS_URL}')
+    cinemas = {}
+    for cinema in cinemas_data:
+        id = cinema['CinemaId']
+        name = cinema['CinemaName']
+        cinemas[id] = Cinema(name, UK_TIMEZONE)
+    return cinemas
+
+
+def get_attributes(experiences: [dict[str, any]]) -> dict[str, any]:
+    attributes = {'format': []}
+    for experience in experiences:
+        if experience['ExternalId'] == 'AD':
+            attributes['audio-described'] = True
+        elif experience['ExternalId'] == 'Event':
+            attributes['event'] = True
+        elif experience['ExternalId'] in ['Baby Pic', 'Baby Cin']:
+            attributes['carers-and-babies'] = True
+        elif experience['ExternalId'] in ['XP', 'Real D 3D', 'IMAX', 'IMAX 2D', 'IMAX 3D', 'LVS', '3D+ HFR']:
```

Comment:
Please split up `IMAX 3D` and `IMAX 2D` into `IMAX` and `2D` / `3D`. Ideally we would also have a way to get all "3D" showings, which would include Real D and 3D+ HFR in that category. Any ideas?